sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -282,7 +282,9 @@ private void setArchitectures(final @NotNull Device device) {
     // TODO: missing usable memory
 
     Device device = new Device();
-    device.setName(getDeviceName());
+    if (!options.isSendDefaultPii()) {
```

Comment:
Shouldn't this be the other way around? i.e. only set the device name if `sendDefaultPii` is enabled?

Reply:
🤦 was a long week...